src/lib/apiTypesV2/Attribute.cpp

```diff
@@ -52,7 +52,8 @@ std::string Attribute::render(ConnectionInfo* ciP, RequestType requestType, bool
     else
     {
       out = "{";
-      out += pcontextAttribute->toJson(true, renderMode, requestType);  // param 1 'true' as it is the last and only element
+      // FIXME PR: renderMode + attrsFormat ... a bit confusing. Should we join these two somehow? 
```

Comment:
I'm not an expert ...
I believe they (renderMode and attrsFormat) are pretty similat, however, renderMode has the possible value "unique"